### PR TITLE
updated EFS-staging transition_to_ia & template file to add multi EFS ref:SAAS-12263

### PR DESCRIPTION
**Pull request by rameshganne**
In the process of v12 validation & testing, I am updating EFS attribute `transition_to_ia`  from staging resource file & removing `{% if not loop.first %}, {% endif %}`  from template file.

### CODE REVIEW

File: environments/staging/terraform.yml [hunk 1]
```diff
@@ -429,7 +429,7 @@ r53_private_zone:
 efs_file_systems:
   - efs_name: "efs_staging"
     create: yes
-    transition_to_ia:  "AFTER_14_DAYS"
+    transition_to_ia:  "AFTER_7_DAYS"
```
Comment (dannyroberts) [inline environments/staging/terraform.yml:432]: Can you explain why this change was necessary for v12?

File: src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2 [hunk 2]
```diff
@@ -313,7 +313,7 @@ module "efs_mount__{{ efs_file_system_mount.efs_name }}__{{ az }}" {
 {% endfor %}
 {% endfor %}
 
-{%- for efs_file_system_route53 in efs_file_systems %}{% if not loop.first %}, {% endif %}
+{%- for efs_file_system_route53 in efs_file_systems %}
```
Comment (dannyroberts) [inline src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2:316]: Can you explain why these changes (removing a comma) were necessary for v12? I'm not sure why they would have ever worked on v11 either 🤔

Comment (rameshganne) [reply src/commcare_cloud/commands/terraform/templates/commcarehq.tf.j2:316]: I had included a new EFS resource to validate module but I could not create it with existing code because adding comma(,) after EFS resource block in commcarehq.tf file. I got know,  `{% if not loop.first %}, {% endif %}` is not required to create multiple EFS resources. To resolve it, I am removing `{% if not loop.first %}, {% endif %}`. 

After changes, I had verified it by adding multiple EFS file-systems.

File: environments/staging/terraform.yml (hunk 1 above)
Comment (rameshganne) [reply environments/staging/terraform.yml:432]: This change does not relate to v12(sorry for using `In the process of v12 validation & testing` in the initial comment). In the general practice & to reduce EFS storage price, I am updating the attribute for moving the files which have not been accessed for the last 7 days to the IA storage class.

**Review by dannyroberts — APPROVED** [review]
